client/my-sites/stats/controller.js

```diff
@@ -99,6 +102,17 @@ module.exports = {
 		}
 	},
 
+	graph: function( context ) {
```

Comment:
Yeah agree, I've not had the time yet to look at how the DevDocs are built etc... so It was quicker for me to have something showing up for now.